api/src/main/java/org/pmiops/workbench/api/ProfileController.java

```diff
@@ -212,6 +215,9 @@ private DbUser initializeUserIfNeeded() {
     if (workbenchConfigProvider.get().captcha.enableCaptcha) {
       verifyCaptcha(request.getCaptchaVerificationToken());
     }
+
+    validateTermsOfService(request.getTermsOfServiceVersion());
```

Comment:
I did not move it with profileService.validateNewProfile(profile); since TOS is not part of PROFILE.
Also if did not make sense to keep it in userService.

Reply:
This looks like the sensible location for it 👍